src/app/partners/page.tsx

```diff
@@ -0,0 +1,41 @@
+import { Header } from "@/components/Header";
+import React from "react";
+import { PartnerCard } from "@/app/partners/PartnerCard";
+import { newApiClient, Organisation } from "@/services/api";
+
+const PartnersPage = async (): Promise<JSX.Element> => {
```

Comment:
It might be fun to look into caching so we don't rerender (and call the api) for every load of this page. I would expect partners to change infrequently so caching the page for an hour could be a nice touch (not that we have anywhere near enough traffic atm to worry)